scripts/baseline_comparison/plot_utils.py

```diff
@@ -0,0 +1,55 @@
+from dataclasses import dataclass
```

Comment:
Because it only works with results produced by the script in `baseline_comparison`, if we have another use-case and make it generic, we could also move it to the main directory.